backend/models/projects/projects-model.test.js

```diff
@@ -11,7 +27,136 @@ afterAll(async () => await db.closeDatabase())
+
+            // Tests
+            expect(createdProject.title).toEqual(expectedTitle)
+            expect(createdProject.status).toEqual(expectedStatus)
+            expect(createdProject.description).toEqual(expectedDescription)
+            expect(createdProject.start_date).toEqual(expectedStartDate)
+            expect(createdProject.end_date).toEqual(expectedEndDate)
+            expect(createdProject.discord_url).toEqual(expectedDiscordUrl)
+            expect(createdProject.github_url).toEqual(expectedGithubUrl)
+            expect(createdProject.demo_url).toEqual(expectedDemoUrl)
+            expect(createdProject.created_at).toEqual(expectedCreatedAt)
+            expect(createdProject.updated_at).toEqual(expectedUpdatedAt)
+        })
+
+        test("Update a Project in Database", async () => {
+            const project = new ProjectsModel(testProject1)
+            await project.save()
+
+            const updatedValues = {
+                title: "Project 1 Updated",
+                status: "Completed",
+                description: "Updated Description",
+                start_date: new Date("12-25-2022"),
+                end_date: new Date("12-25-2023"),
+                discord_url: "https://discord.gg/54321",
+                github_url: "https://github.com/54321",
+                demo_url: "https://demo.com/54322",
+                created_at: new Date("09-27-2022"),
+                updated_at: new Date("09-2-2022"),
+            }
+
+            // Update values
+            project.title = updatedValues.title
+            project.status = updatedValues.status
+            project.description = updatedValues.description
+            project.start_date = updatedValues.start_date
+            project.end_date = updatedValues.end_date
+            project.discord_url = updatedValues.discord_url
+            project.github_url = updatedValues.github_url
+            project.demo_url = updatedValues.demo_url
+            project.created_at = updatedValues.created_at
+            project.updated_at = updatedValues.updated_at
+
+            const updatedProject = await project.save()
+
+            // Tests
+            expect(updatedProject._id).toBeDefined()
+            expect(updatedProject.title).toEqual(updatedValues.title)
+            expect(updatedProject.status).toEqual(updatedValues.status)
+            expect(updatedProject.description).toEqual(
+                updatedValues.description
+            )
+            expect(updatedProject.start_date).toEqual(updatedValues.start_date)
+            expect(updatedProject.end_date).toEqual(updatedValues.end_date)
+            expect(updatedProject.discord_url).toEqual(
+                updatedValues.discord_url
+            )
+            expect(updatedProject.github_url).toEqual(updatedValues.github_url)
+            expect(updatedProject.demo_url).toEqual(updatedValues.demo_url)
+            expect(updatedProject.created_at).toEqual(updatedValues.created_at)
```

Comment:
```suggestion
            expect(updatedProject.created_at).not.toEqual(updatedValues.created_at)
```
Recommend to test an updated `created_at` field with different date to indicate that the content in `create_at` doesn't change. Refer line `110`